firstvoices/backend/resources/utils/import_export_widgets.py

```diff
@@ -105,13 +105,26 @@ def __init__(self, *args, **kwargs):
 
     def clean(self, value, row=None, **kwargs):
         category_match_pattern = r"category[_2-5]*"
-        input_categories = [
-            value
-            for key, value in row.items()
-            if re.fullmatch(category_match_pattern, key)
-        ]
 
+        input_categories = {}
+        valid_categories = []
+
+        for column, input_value in row.items():
+            if re.fullmatch(category_match_pattern, column):
+                input_categories[column] = input_value
+
+        # If no categories provided, return
         if len(input_categories) == 0:
             return Category.objects.none()
 
-        return Category.objects.filter(title__in=input_categories)
+        # Validate categories
+        for column, input_value in input_categories.items():
+            category_lookup = Category.objects.filter(
```

Comment:
[minor] Does this category lookup mean that the category in the csv file needs to match the title of an existing category 1:1? If so would making it case-insensitive have any benefit for the batch-importer?

Reply:
This is a good thought, but currently category uniqueness is case-sensitive. I managed to set up both "Animals" and "animals" as categories on one of the dev sites. 

Let's keep this import code case-sensitive, and I will log a ticket to consider making category uniqueness case insensitive.

Reply:
Yes, in the tech design, it was mentioned to be case-sensitive as to conform with the db model so I kept it that way.
+1 for the ticket to make category case-insensitive.